chevah/github_hooks_server/handler.py

```diff
@@ -56,6 +47,33 @@ def ping(self, event):
             return 'Pong! But GitHub Zen text is missing.'
         return f'Pong! {zen}'
 
+    def pull_request(self, event):
+        """
+        Called, among other cases, when a PR review is requested.
+
+        Triggers the same actions as the needs-review command to issue_comment.
+
+        The `review_requested` action is under the `pull_request` event:
+        https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request
+        """
+        action = event.content.get('action')
+        if action != 'review_requested':
+            logging.warning(f"No handler for pull_request action '{action}'.")
+            return
+
+        title = event.content['pull_request']['title']
+        ticket_id = self._getTicketFromTitle(title)
+
+        repo = event.content['repository']['full_name']
+        pull_id = event.content['pull_request']['number']
+        reviewers = self._getReviewers(event.content['pull_request']['body'])
+
+        logging.info(
```

Comment:
this is ok...but in the future maybe we can also pass the event to _setNeedsReview and have a single log.

I see that _setNeedsReview does it's own extra debug logging.

Reply:
I'll keep it, since we call _setNeedsReview from two places now, and it is useful to know who called it.